### PR TITLE
feature/rainbow-kit

### PR DESCRIPTION
**Pull request by westonnelson**
Added wagmi + RainbowKit for Mainnet, Optimism, Arbitrum.

### CODE REVIEW

**Comment by DistractionBoy** [comment]
for the name of the branch, could you change it to `feature/wallet-connect`?

File: wagmi.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { getDefaultWallets } from '@rainbow-me/rainbowkit'
+import { chain, configureChains, createClient } from 'wagmi'
+import { publicProvider } from 'wagmi/providers/public'
+import { alchemyProvider } from 'wagmi/providers/alchemy'
+
+const { chains, provider } = configureChains(
+ [chain.mainnet, chain.optimism, chain.arbitrum],
+ [alchemyProvider({ apiKey: 'RbvJ2z-poT2yB-cpZ9xUMaauzUeTQVf5' })],
```
Comment (DistractionBoy) [inline wagmi.ts]: this api key should be an environment variable. we should generate a new key to use for production, and maybe another for local / testing

File: wagmi.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      alchemyProvider({
+        apiKey: 'RbvJ2z-poT2yB-cpZ9xUMaauzUeTQVf5',
+        priority: 0,
+      }),
+      publicProvider({priority: 1}),
+   ],
+)
+
+const { connectors } = getDefaultWallets({
+  appName: 'My wagmi + RainbowKit App',
```
Comment (Millsky) [inline wagmi.ts]: Maybe we should have our apps name here?

**Review by Millsky — COMMENTED** [review]
Nice! Just a small nit from my end

File: wagmi.ts (hunk 2 above)
Comment (westonnelson) [reply wagmi.ts]: Good idea! Makes sense to me.

File: wagmi.ts (hunk 1 above)
Comment (westonnelson) [reply wagmi.ts]: Ok - awesome. Yes was thinking we could have one for the frontend and backend and also for staging and production as well!

**Pull request closed without merging**